src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -116,6 +123,17 @@ void fillInnerParts() {
         resultDisplay = new ResultDisplay();
         resultDisplayPlaceholder.getChildren().add(resultDisplay.getRoot());
 
+        // Show internship application details on click
+        ListView<Company> companyListView = companyListPanel.getCompanyListView();
+        companyListView.setOnMouseClicked(new EventHandler<MouseEvent>() {
```

Comment:
This can be replaced with the view function. Or do we want both mouse and view function?